src/main/java/uk/gov/hmcts/reform/sendletter/notify/NotifyClientStub.java

```diff
@@ -1,23 +0,0 @@
-package uk.gov.hmcts.reform.sendletter.notify;
-
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
-import org.springframework.stereotype.Component;
-import uk.gov.hmcts.reform.sendletter.model.Letter;
-
-@Component
-@ConditionalOnProperty(prefix = "notify", name = "stub", havingValue = "true")
```

Comment:
please remove all configuration of `notify.stub`: yml, docker, shell script for docker